README.adoc

```diff
@@ -191,24 +191,14 @@ automatically injected into your pods when you deploy your application.
 
 Navigate to the `start` directory to begin.
 
-The MicroProfile Fault Tolerance API was added as a dependency to your `inventory` service's [hotspot file=0]`pom.xml` file. Look for
-the dependency with the [hotspot=mpFaultTolerance file=0]`mpFaultTolerance` artifact ID. Adding this dependency allows you to use the
-fault tolerance policies in your microservices.
+The MicroProfile Fault Tolerance API is added as a [hotspot=mpFaultTolerance file=0]`feature` in the inventory [hotspot file=0]`server.xml` file.
```

Comment:
- [ ] maybe still keep the dependency part
    - do it like how the other microprofile guides did it, by saying that the API is included in the microprofile dependency
    - e.g. https://github.com/OpenLiberty/guide-microprofile-config/pull/115/files#diff-7bd4a925c695c2eb0eced3872b9b965fL109